nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```
@@ -313,6 +318,15 @@ in {
       '';
     };
 
+    bootCounting = {
+      enable = mkEnableOption (lib.mdDoc "automatic boot assessment");
+      trials = mkOption {
```

This should be called "tries" in line with the upstream documentation. E.g. kernel-install looks in `/etc/kernel/tries`.

```suggestion
      tries = mkOption {
```

---

Done in https://github.com/NixOS/nixpkgs/pull/330017